api/server/router/network/filter.go

```diff
@@ -29,6 +29,29 @@ func filterNetworkByType(nws []types.NetworkResource, netType string) ([]types.N
 	return retNws, nil
 }
 
+func filterNetworkByUse(nws []types.NetworkResource, danglingOnly bool) ([]types.NetworkResource, error) {
+	retNws := []types.NetworkResource{}
+
+	switch danglingOnly {
```

Comment:
That works too, but I don't see why we need to do this?

I mean the code currently tries to be consistent with the existing function filterNetworksByType, and staying consistent would mean better readability. Not that I'm against your suggestion, I just don't see any advantage in doing so.